src/uct/ib/ud/base/ud_iface.h

```diff
@@ -225,6 +227,7 @@ struct uct_ud_ctl_hdr {
             uct_ud_ep_addr_t    ep_addr;
             uct_ud_ep_conn_sn_t conn_sn;
             uint8_t             path_index;
+            uint32_t            dest_ep_id;
```

Comment:
this dest_ep_id is needed to be able to use it for handling CREQ in case if UD endpoint couldn't be found in CEP, but exists in PTR array.

Reply:
we already have dest id in neth, why need here too?

Reply:
to not have another branch which handle CREQ when `neth->packet_type` has real ep_id.
anyway we break backward compatibility by adding new packet type - we have `ucs_fatal` if received unknown packet.
btw, shall we remove this `ucs_fatal` for future?

Reply:
this branch will be not in CREQ handling code, but in `uct_ud_ep_process_rx`.
we will need to check whether it is CREQ or not in case of EP can't be fund in PTR array